dbclient/ClustersClient.py

```diff
@@ -230,10 +234,16 @@ def import_cluster_configs(self, log_file='clusters.log', acl_log_file='acl_clus
             return
         current_cluster_names = set([x.get('cluster_name', None) for x in self.get_cluster_list(False)])
         old_2_new_policy_ids = self.get_new_policy_id_dict()  # dict of {old_id : new_id}
+        error_logger = logging_utils.get_error_logger(
+            wmconstants.WM_IMPORT, wmconstants.CLUSTER_OBJECT, self.get_export_dir())
+        checkpoint_cluster_configs_set = self._checkpoint_service.get_checkpoint_key_set(
```

Comment:
Note that you are reusing the same checkpoint `CLUSTER_OBJECT` for different objects. If the ids happen to be the same, there will be data loss.

Reply:
yeah, this is rather intentional as the ids of these objects cannot be same. (just different id formats to start with)
Didn't want to convolute the number of files. But please let me know if you think the otherwise, can definitely make the change.

Reply:
e.g. one is UUID, the other is cluster path.